pytorch_lightning/utilities/imports.py

```diff
@@ -54,13 +54,11 @@ def _compare_version(package: str, op, version) -> bool:
     except (ModuleNotFoundError, DistributionNotFound):
         return False
     try:
-        pkg_version = LooseVersion(pkg.__version__)
-    except AttributeError:
-        return False
-    if not (hasattr(pkg_version, "vstring") and hasattr(pkg_version, "version")):
+        pkg_version = Version(pkg.__version__)
+    except TypeError:
```

Comment:
@Borda this breaks if the pkg doesn't have `__version__` defined. the previously caught attribute error is now raised. i'm seeing this with torchtext which doesn't seem to export the version info from source: https://github.com/pytorch/text/blob/ccebe7ced8d2229b98e39546fcfbf3e233bd70d9/torchtext/__init__.py#L10-L14

cc @parmeet

Reply:
Well, this seems as `torchtext` as they import own module `version` which is missing... Also their `git_version` is gone